packages/wonder-blocks-toolbar/src/components/__docs__/toolbar.argtypes.js

```diff
@@ -53,6 +70,14 @@ export const rightContentMappings: Mappings = {
             <Button style={buttonStyle}>Next exercise</Button>
         </>
     ),
+    responsive: (
+        <>
+            <Button style={styles.onlyDesktop} kind="secondary">
+                Continue
+            </Button>
+            <Button style={styles.fillContent}>Up next: video</Button>
+        </>
+    ),
```

Comment:
**question:** are these options you can select from a drop down in one of the stories?

Reply:
That's right, you can see it in action in the `Default` example.

Also, I'm reusing these options in the stories: https://github.com/Khan/wonder-blocks/pull/1647/files#diff-075b5d35dd386ef2193679385c0742cb32a4f3bcf4e8a881762b0fb29ab48ac5R171